.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea/
 *.pyc
 *.iml
+logging/
+config_dev.yaml
+passfile.txt
```

Comment:
It would just be the difference between moving the password and username from a separate file to the same config file as all the other access credentials, and a small amount of code rework to use the alternate accreditation method.  I don't think it would alter the security.